### PR TITLE
Stylize GitHub correctly

### PR DESCRIPTION
**Pull request by 0mp**

### CODE REVIEW

**Comment by ShaneHarvey** [comment]
Would you like to fix the other instances of "github"? A quick grep showed some in doc/installation.rst, doc/tools.rst, and CONTRIBUTING.rst.

**Comment by 0mp** [comment]
@ShaneHarvey sure. hang on

**Comment by 0mp** [comment]
Thanks 👍

File: README.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 =======
 PyMongo
 =======
-:Info: See `the mongo site <http://www.mongodb.org>`_ for more information. See `github <http://github.com/mongodb/mongo-python-driver/tree>`_ for the latest source.
+:Info: See `the mongo site <http://www.mongodb.org>`_ for more information. See `GitHub <http://github.com/mongodb/mongo-python-driver/tree>`_ for the latest source.
```
Comment (ShaneHarvey) [inline README.rst]: This link doesn't work anymore. Can you remove the "/tree" from the end?

**Pull request merged**